core/lib/zksync_core/src/lib.rs

```diff
@@ -782,7 +782,7 @@ pub async fn initialize_components(
     }
 
     if components.contains(&Component::ProofDataHandler) {
-        task_futures.push(tokio::spawn(proof_data_handler::run_server(
+        task_futures.push(tokio::spawn(run_server(
```

Comment:
Bikeshedding: IMO, using a qualified name here (i.e., `zksync_proof_data_handler::run_server(..)` would make more sense here; `run_server` is quite ambiguous (lots of components that have their servers, and `lib.rs` concerns all components).